RecoTracker/TkTrackingRegions/interface/GlobalTrackingRegion.h

```diff
@@ -51,6 +51,7 @@ class GlobalTrackingRegion final : public TrackingRegion {
                                               float gz = 0,
                                               float dr = 0,
                                               float dz = 0) const override;
+  std::vector<bool> checkTracks(reco::TrackCollection const& InputCollection) const override;
```

Comment:
do you mind to add the comment above this function as you have in the TrackingRegion.h ?
`/// return a mask over a track collection reflecting the compatability to the region`